src/test/java/api/requests/RequestScheduledNoticesProcessingTests.java

```diff
@@ -710,6 +717,20 @@ private RequestBuilder buildTitleLevelRequest(LocalDate requestExpiration) {
       .withRequestExpiration(requestExpiration);
   }
 
+  private RequestBuilder buildTitleLevelRequest(LocalDate requestExpiration, ItemResource item) {
```

Comment:
Try reolace it with `RequestsFixture.placeTitleLevelPageRequest`

Reply:
done